test/verify/check-service

```diff
@@ -112,12 +112,12 @@ class TestService(composerlib.ComposerCase):
         b.wait_visible("button:contains('Start')")
 
         # disable this test due to https://github.com/osbuild/cockpit-composer/issues/941
```

Comment:
I realize that the issue is still open but I imagine this comment could be really confusing if left in place in a few years when we've all forgotten about the unification...

Reply:
nice catch.